Mathlib/Order/CompleteLattice.lean

```diff
@@ -632,6 +632,9 @@ theorem le_iSup (f : ι → α) (i : ι) : f i ≤ iSup f :=
 theorem iInf_le (f : ι → α) (i : ι) : iInf f ≤ f i :=
   sInf_le ⟨i, rfl⟩
 
+lemma iInf_le_iSup [Nonempty ι] : ⨅ i, f i ≤ ⨆ i, f i :=
+  (iInf_le _ (Classical.arbitrary _)).trans <| le_iSup _ (Classical.arbitrary _)
```

Comment:
Perhaps a bit cleaner:
```suggestion
  let x := Classical.arbitrary ι
  (iInf_le _ x).trans <| le_iSup _ x
```

Reply:
I am aware of this solution but decided against it